src/assets/stylesheets/index.scss

```diff
@@ -859,9 +859,9 @@ section .card.no-color-adaption {
 
 img.img-flag {
   display: inline-block;
-  height: 15px;
   margin: -3px 6px 0 0;
-  max-width: 17px;
+  max-height: 15px;
+  width: 17px;
```

Comment:
This looks better then before, but there is still a short "flicker" when opening up the language menu. Even with the upcoming `1.49.0-rc1`:
<img width="1512" alt="Screenshot 2025-03-31 at 14 32 12" src="https://github.com/user-attachments/assets/cece2519-edf4-45a2-8708-c3de82a88152" />

See full video:

https://github.com/user-attachments/assets/9964a4da-5f22-4059-9d66-dc7f4023b5b0

Reply:
My apologies: I merged the wrong branch. With `1.49.0-rc1` there is no flicker and it looks good.